MK-7462 Update 07-chang.mdx (#650)
input-output-hk/cardano-documentation: docs/about-cardano/05-evolution/04-upgrades/07-chang.mdx

- The `minFeeRefScriptsCoinsPerByte` protocol parameter refines the Plutus cost model and improves fee calculations by allowing separate tuning of the costs for reference scripts.
- Adjustments to SPO deposits: SPO deposits are tracked individually, distinguishing between stake address registration deposits and stake pool registration deposits. This distinction has been retroactively applied all the way back to Shelley, facilitated by the static nature of deposits since their introduction.
- General ledger rules improvements.

## The Chang upgrade in stages

Chang will be gradually implemented in two stages. The first stage established foundational governance features, and the second will emphasize community-driven decision-making and governance.

**First stage/bootstrapping**

The first stage was completed in August, 2024. Its purpose is to enable initial governance features and lay the groundwork for Conway-era functionality. During this time, DReps can register and ada holders can delegate their voting power. PlutusV3 was introduced, which included new governance features.

> An interim constitution was established with safeguards supported by a technical guardrails script. An interim constitutional committee will oversee the constitutionality of governance actions during the interim period, prior to ratification of a full constitution. The bootstrap period will be used to form the group of DReps. Limited governance actions will be supported, including parameter changes, hard forks, and 'Info' actions.

Reviewer: ```suggestion
An interim constitution was established with safeguards supported by a technical guardrails script. An interim constitutional committee oversees the constitutionality of governance actions during the interim period, prior to ratification of a full constitution. The bootstrap period is used to form the group of DReps. Limited governance actions are supported, including parameter changes, hard forks, and 'Info' actions.
```

> During the interim period, only SPOs and the interim constitutional committee can vote on governance actions, including possible protocol parameter changes. As an exception, DReps will can participate in 'Info' actions at this stage.

Reviewer: ```suggestion
During the interim period, only SPOs and the interim constitutional committee can vote on governance actions, including possible protocol parameter changes. As an exception, DReps can participate in 'Info' actions at this stage.
```
